scripts/go-lint-all.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -eu
+set +e
```

Comment:
We should not be removing `-u`. This is `-e` that lets the bash script exit immediately.
If we do not exit immediately, and the last linting command passes, what will be the exit code?

Reply:
nice.  OK, let's see about tossing a -u in there, in addition to the +e

Reply:
https://github.com/cosmos/cosmos-sdk/actions/runs/4566378826/jobs/8058795316

it fails

Reply:
It fails because the last linter is failing. I've just tried with a small script, and the exit code is 0.
We either need to keep exiting immediately, or store if an exit code isn't 0.

```sh
#!/usr/bin/env bash

rm doesnotexit
echo "foo"
```

```sh
$ ./foo.sh
rm: cannot remove 'doesnotexit': No such file or directory
foo
$ echo $?
0
```

Reply:
ahhhhh yes should fail if any fails

Reply:
Best to just leave it as it imo, and do a lint fix all as you did here: https://github.com/cosmos/cosmos-sdk/pull/15631.
Once all the linting will be fixed this won't be a problem.

Reply:
decent pint